Fix:
./configure: line 4851: ++: command not found
./configure: line 5418: ++: command not found
--- configure.orig	2008-07-28 20:12:13.000000000 -0500
+++ configure	2022-01-17 06:16:24.000000000 -0600
@@ -4848,7 +4848,6 @@
 
 
 am_cv_prog_cc_stdc=$ac_cv_prog_cc_stdc
-++
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
 ac_compile='$CC -c $CFLAGS $CPPFLAGS conftest.$ac_ext >&5'
@@ -5415,7 +5414,6 @@
 _ACEOF
 
 fi
-++
 # Check whether --enable-shared was given.
 if test "${enable_shared+set}" = set; then
   enableval=$enable_shared; p=${PACKAGE-default}
